src/geodesic-geom.ts

```diff
@@ -67,13 +67,13 @@ export class GeodesicGeometry {
     multiLineString(latlngs: L.LatLng[][]): L.LatLng[][] {
         const multiLineString: L.LatLng[][] = [];
 
-        latlngs.forEach((linestring) => {
+        for (let linestring of latlngs) {
```

Comment:
Thanks for merging my PR, I'm gonna be back with more stuff soon.

Sure, we can change that to const. Just a matter of preference. Personally, I gave up on constants, because sometimes it appears that a value does need to be changed. Not fun to change const to let back and forth.